src/test/java/com/ibm/cloudant/kafka/connect/CloudantSourceTaskTest.java

```diff
@@ -41,7 +41,6 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
```

Comment:
This removal made me notice the dependency on `com.google.common.collect.Streams` - which I think is transitive as I can't see it declared anywhere (it probably comes via GSON).

We should probably just replace the usages of `Streams.stream(data)` with `java.util.stream.StreamSupport.stream(data.spliterator(), false)` as I don't think there is anything specific about the google commons one needed.

It'd be a bit of scope creep here, please can you open an issue to follow it up?